src/parquet/column_reader.cc

```diff
@@ -51,6 +51,12 @@ ColumnReader::ColumnReader(const parquet::ColumnMetaData* metadata,
     case CompressionCodec::SNAPPY:
       decompressor_.reset(new SnappyCodec());
       break;
+    case CompressionCodec::LZO:
+      decompressor_.reset(new Lz4Codec());
+      break;
```

Comment:
Hmm, I tried enabling Lz4Codec this way on the current trunk and it crashed. I think more work is needed to enable lzo support.